README.md

```diff
@@ -94,7 +94,7 @@ cond, vals, err := builder.NamedQuery("select * from tb where name={{name}} and
 	"m_score": []float64{3.0, 5.8, 7.9},
 })
 
-assert.Equal("select * from tb where name=? and id in (select uid from anothertable where score in (?,?,?))", cond)
+assert.Equal("select * from tb where name=? and id in (select uid from anothertable where score in (?, ?, ?))", cond)
```

Comment:
The line isn't document, it's the real result returned from BuildSelect, and there's no space between `?,`